src/DetailsView/Styles/detailsview.scss

```diff
@@ -228,6 +231,74 @@ div.insights-file-issue-details-dialog-container {
         margin-top: 30px;
         font-size: 21px;
     }
+    .footer {
+        margin-bottom: 16px;
+        margin-top: 8px;
+        padding-top: 16px;
+        padding-right: 16px;
+        border-top-style: solid;
+        border-top-width: 1px;
+        border-top-color: #edebe9;
+    }
+}
+
+.learn-more {
+    color: $communication-primary;
+    text-decoration: none;
+    display: inline-block;
+    margin-bottom: 24px;
+}
+
+.failure-instance-selector {
+    .ms-Label {
```

Comment:
As above:

can we get this styling without using internal office fabric classes? while these may be unlikely to change, since they have no obligation to keep that consistent, we open up our CSS to being broken in that scenario. I would only do this if we were certain there is no other way.

Reply:
Looked like this one in particular wasn't actually doing anything so I removed it/checked for other instances

Reply:
Same as above^